.clang-format

```
@@ -22,5 +22,7 @@ IncludeCategories:
     Priority: 3
 PointerAlignment: Left
 SortIncludes: true
+---
+Language: Json
```

```suggestion
Language: Json
IndentWidth: 4
```

Many of the changes in this patch come down to indentation. Could we add an entry to this file to reflect the current state?

---

That is possible, but perhaps indenting with 2 columns may be better for consistency with most source files, which are in C++.
If you still prefer to maintain the current indent width for Json, please let me know; I'll apply the suggested change.

---

The maximum indent depth of any of the JSON files we have is 5 levels. With 4 spaces, that would be 20 columns, probably more than we want. I suppose 2 columns is okay. But, let's make it explicit. Do you know how the `IndentWidth` was determined in your suggestion? Was it inherited from the prior language, or some invisible default?

---

My current understanding is that it falls back to the LLVM style, which sets `IdentWidth` to 2.

---

OK, this seems fine. We can always update it later if we want. Please update `.github/workflows/main.yml` so that this is enforced across the codebase.

---

I updated that file. Adding `test` to format JSON files in that directory also caused many C++ files to be formatted.